octopoes/octopoes/core/service.py

```diff
@@ -129,10 +129,18 @@ def _run_inference(self, origin: Origin, valid_time: datetime):
         parameters = self.ooi_repository.get_bulk({x.reference for x in parameters_references}, valid_time)
 
         try:
-            resulting_oois = BitRunner(bit_definition).run(source, list(parameters.values()))
+            try:
+                level = self.scan_profile_repository.get(source.reference, valid_time).level
+            except ObjectNotFoundException:
+                level = 0
+            if level >= 1:
+                resulting_oois = BitRunner(bit_definition).run(source, list(parameters.values()))
+            else:
+                resulting_oois = []
```

Comment:
This would be better:
```python3
try:
    level = ...
    if level > 0:
        resulting_oois = ...
except ...:
    resulting_oois = []
```

Reply:
I refactored this part a bit, better?

Reply:
The condition check wasn't the issue, but the code blocks